dotcom-rendering/src/model/enhanceTableOfContents.ts

```diff
@@ -4,8 +4,10 @@ import type { TableOfContentsItem } from '../types/frontend';
 
 const isH2 = (element: CAPIElement): element is SubheadingBlockElement => {
```

Comment:
This return type should have been changed to `element is SubheadingBlockElement | NumberedTitleBlockElement`